docs/developer/contributing.md

```diff
@@ -43,8 +43,16 @@ and
 Once you're ready for review make sure the tests pass:
 
     $ pip install -e .[dev]
+    $ pip install -r dfvfs_requirements.txt
     $ python ./run_tests.py
 
+
+----
+>  **_NOTE:_** If you are developing in a hybrid/local setup, you need to 
+modify the `PROMETHEUS_PORT` for the server in `turbinia/client.py` in order
```

Comment:
I have a PR to change the code not to load prometheus if the config is set to None.  Maybe specifically mention setting that to None here?